packages/components/src/style-provider/index.tsx

```diff
@@ -3,7 +3,6 @@
  */
 import { CacheProvider } from '@emotion/react';
 import createCache from '@emotion/cache';
-import memoize from 'memize';
```

Comment:
Memoize should never be used for anything except if the max size is 1 imo. Always creates memory problems. Maybe we should add a rule to at least always define the maxSize option.

Reply:
> A maxSize of 1000 will still unnecessarily leak at most 1000 detached documents, which is still a lot.

as @kevin940726 noted, this still leaves ample room for memory bloat, but I think it's better than uncapped memoization. maybe we can look into places where we use it and analyze each situation to figure out a reasonable cap.